### PR TITLE
Fixed migration that caused errors on test database

### PR DESCRIPTION
**Pull request by Larspolo**
This change is [Reviewable](https://reviewable.io/reviews/ejourn-al/ejournal/1309)

### CODE REVIEW

File: src/django/VLE/migrations/0104_moved_preset_only_to_template_chain.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 def transfer_preset_only_to_template_chain(apps, schema_editor):
     TemplateChain = apps.get_model('VLE', 'TemplateChain')
     Template = apps.get_model('VLE', 'Template')
-    for template_chain in TemplateChain.objects.all():
+    for template_chain in TemplateChain.objects.filter(template__isnull=False):
```
Comment (mjvkeulen) [inline src/django/VLE/migrations/0104_moved_preset_only_to_template_chain.py:9]: How about we remove isolated TemplateChain instances instead?

Comment (Larspolo) [reply src/django/VLE/migrations/0104_moved_preset_only_to_template_chain.py:9]: This worked, dont want to spend more time on it, you may improve it if you want

Comment (mjvkeulen) [reply src/django/VLE/migrations/0104_moved_preset_only_to_template_chain.py:9]: So what happens with isolated TemplateChain instances in the future?

Comment (mjvkeulen) [reply src/django/VLE/migrations/0104_moved_preset_only_to_template_chain.py:9]: These should not exist -> just remove them

Comment (Larspolo) [reply src/django/VLE/migrations/0104_moved_preset_only_to_template_chain.py:9]: Think that would also not solve the issue for future instances, so I created this issue https://ejournal.atlassian.net/browse/EJ-1435

**Review by engelhamer — APPROVED** [review]

**Review by mjvkeulen — APPROVED** [review]

**Pull request merged**